src/k8s-engine/images/mobyImageProcessor.ts

```diff
@@ -0,0 +1,131 @@
+import { spawn } from 'child_process';
+import os from 'os';
+import path from 'path';
+
+import Logging from '@/utils/logging';
+import resources from '@/resources';
+import * as imageProcessor from '@/k8s-engine/images/imageProcessor';
+import mainEvents from '@/main/mainEvents';
+import paths from '@/utils/paths';
+import * as K8s from '@/k8s-engine/k8s';
+import * as window from '@/window';
+
+const console = Logging.images;
+
+export default class MobyImageProcessor extends imageProcessor.ImageProcessor {
+  constructor(k8sManager: K8s.KubernetesBackend, label: string) {
+    super(k8sManager, label);
+
+    mainEvents.on('k8s-check-state', (mgr: K8s.KubernetesBackend) => {
+      // There's no need to install kim when using moby, so don't.
+      this.isK8sReady = mgr.state === K8s.State.STARTED;
+      this.updateWatchStatus();
+    });
+  }
+
+  protected get processorName() {
+    return 'moby';
+  }
+
+  protected async runImagesCommand(args: string[], sendNotifications = true): Promise<imageProcessor.childResultType> {
+    const subcommandName = args[0];
+    const dockerArgs = os.platform().startsWith('win') ? args : ['--host', `unix://${ path.join(paths.lima, '0', 'sock', 'docker') }`].concat(args);
```

Comment:
We'll hopefully not need this in the future?
```suggestion
    const dockerArgs = args.concat();
    if (!os.platform.startsWith('win')) {
      dockerArgs.unshift('--host', `unix://${path.join(paths.lima, '0', 'sock', 'docker')}`);
    }
```

Reply:
Yes, this should disappear when https://github.com/rancher-sandbox/rancher-desktop/issues/939 is done. Added a `// TODO` comment.